README.md

```diff
@@ -4,20 +4,29 @@ You can deploy 1Password SCIM Bridge on any supported infrastructure that allows
 
 ## Automatic (one-click) deployment
```

Comment:
I would prefer that we standardize on "marketplace apps" for these deployment options:

```suggestion
## Marketplace apps
```

Reply:
I love this suggestion!